src/pages/{StoreCollection.slug}.tsx

```diff
@@ -1,4 +1,5 @@
-import { parseSearchState, SearchProvider, useSession } from '@faststore/sdk'
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
Is this still needed? I removed it locally but got no ESLint warning, not even when trying to commit and `eslint --fix` was run.

Reply:
I forgot to remove, hehe